contributing/code/standards.rst

```diff
@@ -47,30 +47,24 @@ short example containing most features described below::
      */
     class FooBar
     {
-        const SOME_CONST = 42;
+        public const SOME_CONST = 42;
 
-        /**
-         * @var string
-         */
-        private $fooBar;
-
-        private $qux;
+        private string $fooBar;
+        private Qux $qux;
```

Comment:
We can't use property types on the 5.4 branch, can we?

Reply:
IIRC we can for newly introduced ones

Reply:
Right, they aren't supported in PHP 7.2 of course.